lib/streamlit/elements/arrow.py

```diff
@@ -82,12 +82,12 @@ def dataframe(
             pandas styling features, like bar charts, hovering, and captions.
 
         width : int or None
-            Desired width of the dataframe expressed in pixels. If None, the width
-            will be automatically calculated based on the column content.
+            Desired width of the dataframe expressed in pixels. If None (default),
+            selects the width automatically.
```

Comment:
```suggestion
            Desired width of the dataframe expressed in pixels. If ``width`` is ``None`` (default),
            Streamlit selects the width automatically.
```

One thing I've been grappling with recently is standardizing the grammar within docstrings. We have a mix of fragments and complete sentences and I want to consciously draw a line for when we use each. I've been kicking around the idea that we let the first "sentence" be a noun phrase, stated as a definition, but then switch to complete sentences thereafter if we're going to describe behavior or cases.

I'm a little hesitant that that might be too extreme and get too wordy when applied broadly, so I'm curious about other opinions here. (As an aside, I also want to make sure Python objects are formatted as code. This isn't consistent everywhere yet, but I'm inclined to use `None` in code format.)

When I poke around other libraries, I don't see a lot of consistency elsewhere. Our docstrings tend to be comparatively more complex with a lot more switching behavior in our parameters, too.

Reply:
I also need to have a separate discussion about "dataframe" vs "DataFrame" since pandas, Arrow, and Spark all use the capitalized form. It'd be nice to have an uncapitalized, generic version (and we mostly do in the docs), but I've had a hard time finding anyone else doing that in any official documentation. Maybe we can just choose to lead the charge. 😅

Reply:
I think there was some discussion some time ago related to dataframe vs DataFrame. I believe the outcome was that we only use DataFrame if we are referring to a specific implementation of a dataframe, e.g. Pandas DataFrame, Snowpark DataFrame. But if we are talking about this topic more high-level or if we are talking about the actual table UI we are using lowercased dataframe version. But not sure if we ever made a decision on that.

Reply:
The suggested docstring sounds good to me 👍 and it would be awesome to have this kind of general guidelines on how we write / structure docstrings